frontend/src/containers/LearnMore.js

```diff
@@ -0,0 +1,147 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import LoginTopBar from '../containers/LoginTopBar';
+import PublicFooter from '../components/PublicFooter';
+import Grid, { Column } from '../components/Grid';
+
+export class LearnMore extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className='LearnMore'>
+          <LoginTopBar />
+          <div className='LearnMore-container'>
+            <section className='LearnMore-Section1'>
+              <div className='LearnMoreHeader'>Discover a new form of association</div>
+              <div className='LearnMoreHeader -faded'>Transparent by design.</div>
+              <div className='-img'></div>
+            </section>
+            <section className='LearnMore-Section2'>
+              <div className='LearnMoreHeader'>Open Your Finances To Your Community</div>
+              <Grid className='Wrap900'>
+                <Column span='6'>
+                  <div className='LearnMoreLead'>Unpaid Expenses</div>
+                  <div className='LearnMoreSubLead'>All submited expenses can bee seen by everyone and be marked as “pending” until you approve or reject them.</div>
+                  <div className='-img-1'></div>
+                </Column>
+                <Column span='6'>
+                  <div className='LearnMoreLead'>Funds Avaliable</div>
+                  <div className='LearnMoreSubLead'>Your funds will be openly displayed, everyone will be able to see the money that went in & out of the collective.</div>
+                  <div className='-img-2'></div>
+                </Column>
+              </Grid>
+              <div className='-img'></div>
+            </section>
+            <section className='LearnMore-Section3'>
+              <Grid className='Wrap900'>
+                <Column span='7'>
+                  <div className='LearnMoreHeader'>Submit Expenses</div>
+                  <div className='LearnMoreText'>Take a picture of the receipt with your phone or upload a PDF of the invoice and submit it to the collective.</div>
```

Comment:
"Take a picture of the receipt with your phone or upload a PDF to the collective"